modules/extensions/src/main/java/org/apache/synapse/mediators/bsf/CommonScriptMessageContext.java

```diff
@@ -291,23 +291,40 @@ public Object getEnvelopeXML() throws ScriptException {
     }
 
     /**
-     * Helpers to set EPRs from a script string.
+     * This is used to set the value which specifies the receiver of the message.
+     *
+     * @param reference specifies the receiver of the message
      */
     @Override
     public void setTo(String reference) {
         mc.setTo(new EndpointReference(reference));
     }
 
+    /**
+     * This is used to set the value which specifies the receiver of the faults relating to the message.
+     *
+     * @param reference specifies the specifies the receiver of the faults relating to the message
```

Comment:
Lets remove extra "specifies the" .

Reply:
Hi,
Fixed with https://github.com/wso2/wso2-synapse/pull/971